po/pt_BR.po

```diff
@@ -1922,7 +1919,7 @@ msgstr "versão de Atom inválida"
 
 #: rss/parser.cpp:367
 msgid "no Atom version"
-msgstr "Atom sem versão"
+msgstr "sem versão ATOM"
```

Comment:
```suggestion
msgstr "sem versão Atom"
```

"RSS" is an abbreviation, but "Atom" is not, so no need to make all the letters uppercase.

Reply:
OK